lib/class-wp-theme-json-gutenberg.php

```diff
@@ -785,6 +789,9 @@ protected static function sanitize( $input, $valid_block_names, $valid_element_n
 			$schema_styles_blocks[ $block ]               = $styles_non_top_level;
 			$schema_styles_blocks[ $block ]['elements']   = $schema_styles_elements;
 			$schema_styles_blocks[ $block ]['variations'] = $schema_styles_variations;
+			$schema_styles_blocks[ $block ]['.current-item'] = $styles_non_top_level;
```

Comment:
The value `.current-item` should come from a whitelist of allowed "state selectors". The terminology we use here will be critical.